modules/scf/gen_config.sh

```diff
@@ -112,6 +112,13 @@ properties:
           slow_spec_threshold: 300
           nodes: 2
           flake_attempts: 4
+  brain-tests:
+    acceptance-tests-brain:
+      acceptance_tests_brain:
+        verbose: ${BRAIN_VERBOSE}
```

Comment:
Nitpicking: this vars aren't quoted because it's not needed, yet maybe it's a good idea to quote them

Reply:
Sure. Should not hurt.

Reply:
Addressed with commit 78b25e6730a5ff609b295d14b48e5b8953fe9a05